### ISSUE TITLE
Restriction using throw statement.

### ISSUE THREAD

**Issue by sourceforge-issue-exporter** [beanshell/beanshell#151]
Hi,

This code in BeanShell works.
throw new Exception\(\);

This code should, but doesn't, because the BeanShell 
throw statement has to be used in tandem with an 
Exception, rather than a Throwable object.
throw new Throwable\(\);

Reported by: amc1

Original Ticket: "beanshell/bugs/30":https://sourceforge.net/p/beanshell/bugs/30

**Comment by nickl-** [comment]
**Closed: duplicate #546**

**Comment by nickl-** [comment]
Resolved.

```java
BeanShell 3.0.0-SNAPSHOT.463
bsh % throw new Exception();
// Error: Target Exception: TargetError : at Line: 1 : in file: <unknown file> : throw new Exception ( ) ;
Caused by: java.lang.Exception
bsh % throw new Throwable();
// Error: Target Exception: TargetError : at Line: 2 : in file: <unknown file> : throw new Throwable ( ) ;
Caused by: java.lang.Throwable
bsh %

try {
    throw new Throwable();
} catch (e) {
    print(e);
}

java.lang.Throwable
--> void

```
